packages/maskbook/src/social-network/PostInfo.ts

```diff
@@ -52,6 +52,7 @@ export abstract class PostInfo {
     readonly decryptedPostContent = new ValueRef<TypedMessage | null>(null)
     /** @deprecated It should appear in the transformedPostContent */
     readonly decryptedPostContentRaw = new ValueRef('')
+    readonly iv = new ValueRef('')
```

Comment:
It should be `ValueRef<null | string>`, don't use an empty string as empty state

Reply:
done